### PR TITLE
Update release notes analytics android 1.2.4

### PR DESCRIPTION
**Pull request by nporter-adbe**

### CODE REVIEW

File: release-notes.md [hunk 1]
```diff
@@ -8,6 +8,16 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 Please take a moment to fill out a [short survey](https://www.surveymonkey.com/r/AEPDocs) on how we can better assist you with enabling Adobe Experience Cloud solutions and services on your mobile apps.
 
+## February 13, 2020
+
+The following updates were made in this release:
+
+### Android Analytics 1.2.4
+
+* Fixed an issue which was causing some hits to be delayed.
```
Comment (rekrishn) [inline release-notes.md]: Please add a comma between "issue" and "which.

File: release-notes.md [hunk 2]
```diff
@@ -8,6 +8,16 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
...
+* Fixed an issue where `AnalyticsResponse` events were not being dispatched even when debug API was enabled.
```
Comment (rekrishn) [inline release-notes.md]: Please add "the" between "when" and "debug"

File: release-notes.md [hunk 3]
```diff
@@ -17,7 +27,7 @@ The following updates were made in this release:
 * Fixed a cursor leak.
 * Fixed an issue where the advertising identifier was duplicated in the response to the `MobileCore.getSDKIdentifiers` API.
 * Added support for overriding internal network stack with customer-provided code.
-* Added a new interface with failure callback, `AdobeCallbackWithError`, which can be used with the `MobileCore.getPrivacyStatus`,  `MobileCore.getSdkIdentities` methods. 
+* Added a new interface with failure callback, `AdobeCallbackWithError`, which can be used with the `MobileCore.getPrivacyStatus`,  `MobileCore.getSdkIdentities` methods.
```
Comment (rekrishn) [inline release-notes.md]: Please replace the comma between `MobileCore.getPrivacyStatus` and `MobileCore.getSdkIdentities` with "and", so like this:
"....with the `MobileCore.getPrivacyStatus` and `MobileCore.getSdkIdentities` methods."

**Comment by nporter-adbe** [comment]
@rekrishn changes made.

**Comment by nporter-adbe** [comment]
@rekrishn changes made

**Pull request merged**